### PR TITLE
Fix for Issue #1, added event playoffs as well so those are kept in 12 hour time

### LINKED ISSUE #1: Doesn't always keep the time in 12 hour format, gets overwritten/updated over in some cases (Firefox)
I get an issue of the times quickly being correctly converted to 12 hour, but then being converted back into 24 hour on this page in Firefox. May be able to work on a fix for this if you're interested.

https://www.hltv.org/events/7524/matches

### PR DESCRIPTION
**Pull request by ethel-dev**
With these changes #1 is fixed, at least in my Firefox testing environment. I also went ahead and added the 12hr logic to the event page because playoffs are happening right now and it's easy to test :-) Thanks for making this extension

### CODE REVIEW

File: event.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    if (targetText.includes("am") || targetText.includes("pm")) {
+        observer.disconnect();
+        return;
+    }
+    
+    updateEventPlayoffTimes();
+});
+
+updateEventPlayoffTimes();
+matchObserver.observe(document.querySelector(".time-time"), { childList: true });
```
Comment (smuke) [inline event.js]: Give observer a unique name to prevent a name collision

File: event.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            e.innerText = `12:${mins} PM`;
+        }
+        else {
+            if (hours < 10) e.innerText = `${hours.substring(1)}:${mins} AM`;
+            else e.innerText = `${hours}:${mins} AM`;
+        }
+    });
+}
+
+const matchObserver = new MutationObserver((mutations, observer) => {
```
Comment (smuke) [inline event.js]: Give observer a unique name to prevent a name collision

**Review by smuke — CHANGES_REQUESTED** [review]
Just a small change :)

**Comment by ethel-dev** [comment]
Got it, doing so now

**Comment by ethel-dev** [comment]
How's that?

**Comment by smuke** [comment]
Thanks! Will update on Firefox Add-ons.

**Pull request merged**